pkgs/development/python-modules/types-pyopenssl/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, cryptography
+}:
+
+buildPythonPackage rec {
+  pname = "types-pyOpenSSL";
```

Comment:
`pname` should be normalized according to PEP-503, see also https://nixos.org/manual/nixpkgs/stable/#contributing

```suggestion
  pname = "types-pyopenssl";
```

Reply:
it doesn't find the package on pypi when normalized